docs/detections/rules-ui-manage.asciidoc

```diff
@@ -117,7 +122,16 @@ You can duplicate, enable, disable, and delete rules:
 [[import-export-rules-ui]]
 === Export and import rules
 
-You can export detection rules to an `.ndjson` file, which you can then import into another {elastic-sec} environment. The following configuration items are also included in the `.ndjson` file:
+You can export custom detection rules to an `.ndjson` file, which you can then import into another {elastic-sec} environment. 
+
+[NOTE]
+====
+You cannot export Elastic prebuilt rules, but you can duplicate a prebuilt rule and then export the duplicated rule.
+
+If you try to export with both prebuilt and custom rules selected, only to the custom rules are exported.
```

Comment:
`only to the custom rules are exported` - perhaps `to` is a typo

Reply:
Good catch, thanks!
```suggestion
If you try to export with both prebuilt and custom rules selected, only the custom rules are exported.
```